cogip/tools/server_beacon/robot.py

```diff
@@ -76,25 +76,25 @@ async def disconnect():
             logger.info(f"Beacon handler: disconnected from robot {self.robot_id}")
             await self.server.sio.emit("del_robot", self.robot_id, namespace="/dashboard")
 
-        @self.sio.event(namespace="/dashboard")
-        async def state(robot_id: int, state: dict[str, Any]):
-            await self.server.sio.emit("state", (robot_id, state), namespace="/dashboard")
+        # @self.sio.event(namespace="/dashboard")
```

Comment:
Is it good to merge commented code ? 

Why not delete it and revert this commit later if we want to retrieve this feature ?

Reply:
Deleting/reverting is always difficult because you first have to find the commit to revert and then the code may
have evolved so revert can lead to conflicts.

However, the main raison is that I am often using this code for debugging purpose.

Added a word about that in the commit message.